data/game_teams_sample.csv

```diff
@@ -0,0 +1,946 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways

```

Comment:
Note that this file is the one that wasn't displaying colors in atom... possible issue with this file?